code/style.css

```diff
@@ -1,7 +1,209 @@
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Karla", sans-serif;
+  text-decoration: none;
+  overflow-x: hidden;
+}
+
+/* Header with video hero */
+video {
+  object-fit: cover;
+  width: 100vw;
+  height: 60vh;
+  position: absolute;
+  z-index: -1;
+}
+
+.hero {
+  text-align: center;
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+  text-shadow: 1px 1px 2px #716459;
+}
+
+.hero h1 {
+  font-family: "DM Serif Display", serif;
+  font-style: italic;
+  font-size: 3.5rem;
+  max-width: 80vw;
+}
+
+.hero p {
+  font-size: 1.5rem;
+  max-width: 80vw;
+  font-weight: 400;
+}
+
+/* Section for form and accordion*/
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 2rem;
+  background: #fff7ee;
+  width: 100vw;
+  font-size: 18px;
+}
+
+/* Form and submit button*/
+.form {
+  background: #716459;
+  max-width: 30rem;
+  padding: 2rem;
+  position: relative;
+  width: 100%;
+  color: #fff;
+}
+
+.form form .input-box {
+  width: 100%;
+  height: 2rem;
+  padding: 0 0.5rem;
+  margin: 0.5rem 0 1rem;
+  border: 1px solid black;
+}
+
+.form h2 {
+  margin: 0 0 2rem;
+  text-align: center;
+  font-size: 1.2rem;
+}
+
+a:link, a:visited {
+  color: #fff7ee;
+}
+
+a:hover, a:active {
+  color: #ebe2d7;
+}
+
+.submit-btn {
+  height: 3rem;
+  padding: 0 1rem;
+  margin: 1rem 0 0 0;
+  float: right;
+  font-size: 18px;
+  border: 1px solid black;
+  cursor: pointer;
+  background-color: #fff7ee;
+}
+
+.submit-btn:hover {
+  background: #ebe2d7;
+  transition: all 150ms ease-in;
+}
+
+/* Accordion FAQ */
+.accordion {
+  background: #716459;
+  max-width: 30rem;
+  padding: 2rem;
+  position: relative;
+  width: 100%;
+}
+
+.accordion h2 {
+  margin: 0 0 2rem;
+  text-align: center;
+  color: #fff;
+  font-size: 1.2rem;
+}
+
+.question {
+  background-color: #fff7ee;
+  color: #262523;
+  cursor: pointer;
+  padding: 1rem;
+  width: 100%;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.4s;
+}
+
+.active, .question:hover {
+  background-color: #ebe2d7;
+}
+
+.answer {
+  padding: 0 1rem;
+  background-color: #fff;
+  display: none;
+  overflow: hidden;
+}
+
+.answer p {
+  padding: 1rem 0;
+}
+
+.question:after {
+  content: '\25B8';
+  font-size: 13px;
+  color: #777;
+  float: right;
+  margin-left: 5px;
+}
+
+.active:after {
+  transform: rotate(90deg);
+}
+
+/* Footer */
+footer {
+  background-color: #716459;
+  text-align: center;
+  padding: 1rem;
+}
+
+footer p {
+  color: #fff;
+  font-size: 0.8rem;
+}
 
-.active + .answer {
-  /* some styling here */
+/* Media query tablet */
+@media (min-width: 668px) {
```

Comment:
Your css for the media query (but also css-file) is so easy so pleasant to look at. It's good that you don't repeat yourself within the different breakpoints.